### PR TITLE
Handle gds.alpha.graph.nodeLabel.write

### PR DESCRIPTION
**Pull request by vnickolov**
Thank you for your contribution to the Graph Data Science Client project.

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [ ] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [ ] Your contribution is covered by tests

### CODE REVIEW

**Comment by vnickolov** [comment]
[[WriteLabels] Implement procedure that writes back label to Neo4j](https://trello.com/c/N4g9VDy5/7316-writelabels-implement-procedure-that-writes-back-label-to-neo4j)

File: graphdatascience/graph/graph_proc_runner.py [hunk 1]
```diff
@@ -167,6 +168,11 @@ def graphProperty(self) -> GraphPropertyRunner:
         self._namespace += ".graphProperty"
         return GraphPropertyRunner(self._query_runner, self._namespace, self._server_version)
 
+    @property
+    def nodeLabel(self) -> GraphLabelRunner:
+        self._namespace = "gds.alpha.graph.nodeLabel"
```
Comment (adamnsch) [inline graphdatascience/graph/graph_proc_runner.py:173]:
```suggestion
       self._namespace += ".nodeLabel"
```
This makes sure that users cannot leave out `alpha` for example from their method call

File: graphdatascience/tests/unit/test_graph_ops.py [hunk 2]
```diff
@@ -458,6 +458,19 @@ def test_graph_relationship_write(runner: CollectingQueryRunner, gds: GraphDataS
     }
 
 
+@pytest.mark.parametrize("server_version", [ServerVersion(2, 3, 0)])
+def test_graph_nodeLabel_write(runner: CollectingQueryRunner, gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project("g", "*", "*")
+
+    gds.graph.nodeLabel.write(G, "TestLabel", nodeFilter="n.score > 1.0")
```
Comment (adamnsch) [inline graphdatascience/tests/unit/test_graph_ops.py:465]:
```suggestion
    gds.alpha.graph.nodeLabel.write(G, "TestLabel", nodeFilter="n.score > 1.0")
```

File: changelog.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 ## New features
 
+* Added new method `gds.graph.nodeLabel.write` to write back node labels to Neo4j database.
```
Comment (adamnsch) [inline changelog.md:9]:
```suggestion
* Added new method `gds.alpha.graph.nodeLabel.write` to write back node labels to Neo4j database.
```

**Review by adamnsch — COMMENTED** [review]
Looks good, but I would like to see an integration test as well :) Hopefully it should be straightforward to set up for that by following the docs, but please let me know if you need any help!

**Comment by vnickolov** [comment]
> Looks good, but I would like to see an integration test as well :) Hopefully it should be straightforward to set up for that by following the docs, but please let me know if you need any help!

@adamnsch thank you for the review, I will look into adding integration test and will be in touch if I have any difficulties 🙇

**Review by adamnsch — APPROVED** [review]
Looks great! Let's ship it :tada:

**Pull request merged**